src/jspdf.js

```diff
@@ -1214,9 +1214,9 @@ var jsPDF = (function (global) {
       var orientation = typeof height === 'string' && height.toLowerCase();
 
       if (typeof width === 'string') {
-        if (getPageFormat(width.toLowerCase())) {
-          width = getPageFormat(width.toLowerCase())[0];
-          height = getPageFormat(width.toLowerCase())[1];
+        if (tmp = getPageFormat(width.toLowerCase())) {
```

Comment:
this is not a good comparison. It will check if the variable tmp will be set with the method getPageFormat. So if you supply an invalid pageFormat, e.g. 'l', you will get undefined for the method call and then you will have false for the if clause. Why is this better than the current method?

Reply:
in my opinion `if (getPageFormat(width.toLowerCase()))`
and `if (tmp = getPageFormat(width.toLowerCase()))`
fully equivalent operations, but do not require a triple call
`getPageFormat(width.toLowerCase())`
but the error was in 3 call `width.toLowerCase()` on integer width